pom.xml

```diff
@@ -10,15 +9,37 @@
 	</parent>
 	<groupId>cist4830.unomaha</groupId>
 	<artifactId>tempo</artifactId>
-	<version>0.0.2</version>
+	<version>0.0.3</version>
```

Comment:
Should this be version 0.0.4 since this is milestone 4?

Reply:
hmm, we could honestly call it 1.0.0 i think, since milestone 4 ought to be when we are done

Reply:
We can change it to whatever.  The meat of this PR was just the site/docs.